src/main/java/fi/aalto/cs/apluscourses/intellij/actions/ShowFeedbackAction.java

```diff
@@ -29,7 +30,7 @@
 public class ShowFeedbackAction extends AnAction {
   public static final String ACTION_ID = ShowFeedbackAction.class.getCanonicalName();
 
-  private static final String[] SUPPORTED_COURSES = new String[] {"O1", "Programming Studio 2/A"};
+  public static final List<String> SUPPORTED_COURSES = List.of("O1", "Programming Studio 2/A");
```

Comment:
I feel that hardcoding names of courses in the code is the wrong approach. However, since this PR is only being merged into another branch, I'll leave the discussion about this to be continued in that other PR.

Reply:
I think it should be the course config file that tells which features it uses. Even if this list was moved to resources or wherever, maintaining it would be challenging.

Furthermore, changing type to `List` suggests that this list is perhaps modified run-time, which it shouldn't be. I'd keep it as an array unless there's a good reason to have it as a list.